README.md

```diff
@@ -30,7 +33,8 @@ Parameters:
 + settings - the parameters that will be transformed to the actual service's settings specified in the **services.xml** file.
 + maxMessageBatchSize - the limitation for message batch size which connect sends to the first and to the second publish pins with. The default value is set to 100.
 + enableMessageSendingEvent - if this option is set to `true`, connect sends a separate event for every message sent which incomes from the pin with the send attribute. The default value is set to true
-+ sessionGroup - parameter will be set for all messsages received or sent by this component
++ sessionGroup - parameter will be set for all messages received or sent by this component
++ dictionaries - (optional) dictionaries will be picked up based on its aliases
```

Comment:
Could you please describe about fallback when dictionaries field isn't specified